openapi.yaml

```diff
@@ -1,279 +1,678 @@
+        clientCredentials:
+          tokenUrl: https://api.oregonstate.edu/oauth2/token
+          scopes:
+            full: Full access to the API
+  responses:
+    '400':
+      description: Bad request
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResult'
+    '404':
+      description: Not found
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResult'
+    '409':
+      description: Conflict
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResult'
+    '500':
+      description: Internal server error
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResult'
+  schemas:
+    LocationResult:
+      properties:
+        links:
+          $ref: '#/components/schemas/Links'
+        data:
+          $ref: '#/components/schemas/LocationsResourceObject'
+        included:
+          type: array
+          items:
+            $ref: '#/components/schemas/ServicesResourceObject'
+    LocationSetResult:
+      properties:
+        links:
+          $ref: '#/components/schemas/Links'
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/LocationsResourceObject'
+        included:
+          type: array
+          items:
+            $ref: '#/components/schemas/ServicesIncluded'
+    ServicesResultObject:
+      properties:
+        links:
+          $ref: '#/components/schemas/Links'
+        data:
+          $ref: '#/components/schemas/ServicesResourceObject'
+    ServicesIncluded:
+      properties:
+        id:
+          type: string
+        type:
+          type: string
+        attributes:
+          $ref: '#/components/schemas/ServicesAttributes'
+        links:
+          $ref: '#/components/schemas/ServiceLinks'
+    LocationsResourceObject:
+      properties:
+        id:
+          type: string
+        type:
+          type: string
+        attributes:
+          $ref: '#/components/schemas/Attributes'
+        links:
+          $ref: '#/components/schemas/SelfLink'
+        relationships:
+          $ref: '#/components/schemas/RelationshipsInLocations'
+    ServicesResultObjects:
+      properties:
+        links:
+          $ref: '#/components/schemas/Links'
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/ServicesResourceObject'
+    ServicesResourceObject:
+      properties:
+        id:
+          type: string
+        type:
+          type: string
+        attributes:
+          $ref: '#/components/schemas/ServicesAttributes'
+        links:
+          $ref: '#/components/schemas/ServiceLinks'
+        relationships:
+          $ref: '#/components/schemas/RelationshipsInServices'
+    SelfLink:
+      properties:
+        self:
+          type: string
+          description: Url to this resource / location.
```

Comment:
Unless we missed any filed on the skeleton, I recommend directly use [whatever we've defined there](https://github.com/osu-mist/express-api-skeleton/blob/openapi3/openapi.yaml#L192-L197) instead of defining yourself again. By doing that, we can make sure all of our APIs follow the same style/standard and it will also be easier for us if we need to fix a bug.